cpu/stm32_common/ldscripts/stm32_common.ld

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup      cpu_stm32_common
+ * @{
+ *
+ * @file
+ * @brief           Memory definitions for the STM32 family
+ *
+ * @author          Francisco Acosta <francisco.acosta@inria.fr>
+ *
+ * @}
+ */
+
+boot_offset = DEFINED( offset ) ? offset : 0x0 ;
+ccmram_length = DEFINED( ccmram_len ) ? ccmram_len : 0x0 ;
+
+MEMORY
+{
+    rom (rx)    : ORIGIN = 0x08000000 + boot_offset, LENGTH = rom_length
```

Comment:
If you subtract boot offset from the length you will have one less source of mistakes in the makefiles.